pages/linux/aurman.md

```diff
@@ -0,0 +1,37 @@
+# aurman
+
+> An Arch Linux utility to build and install packages from the Arch User Repository.
+> See also `pacman`.
+> More information: <https://github.com/polygamma/aurman>.
+
+- Synchronize and update all packages:
+
+`aurman --sync --refresh --sysupgrade`
+
+- Synchronize and update all packages without show changes of `PKGBUILD` files:
+
+`aurman --sync --refresh --sysupgrade --noedit`
+
+- Install a new package:
+
+`aurman --sync {{package_name}}`
+
+- Install a new package without show changes of `PKGBUILD` files:
+
+`aurman --sync --noedit {{package_name}}`
+
+- Install a new package without prompting:
+
+`aurman --sync --noedit --noconfirm {{package_name}}`
+
+- Search the package database for a keyword from the official repositories and AUR:
+
+`aurman --sync --search {{keyword}}`
+
+- Remove a package and its dependencies:
+
+`aurman --remove --recursive --nosave {{package_name}}`
+
+- Remove not installed packages and unused repositories from the cache (use two `--clean` flags to clean all packages):
```

Comment:
`Remove not installed packages` doesn't sounds very intuitive at first here. Is there another way we could phrase this?

Reply:
@navarroaxel How about `Remove packages that are no longer installed from the cache as well as currently unused sync databases`? That's what the page referred to in `aurman --help` says.

Reply:
Close, but not exactly the AUR packages are git repositories with a `PKGBUILD` file to build your own package. So `aurman` keeps some builds and that cloned Git repos in `.cache/aurman/`, so this command:
- Remove AUR cloned Git repositories from `aurman` cache for packages that are no longer installed.
- Remove downloaded packages that are no longer installed from the `pacman` cache.
- Also, the unused sync databases are removed but is not important as the previous ones.

Reply:
What about something else, like `Clear the package cache`?

Reply:
Why not, is a more general description but we can explain every step of a command.

Reply:
Sounds good!